Octokit.Tests/Helpers/Arg.cs

```diff
@@ -53,6 +53,21 @@ public static NewRepository NewRepository
             get { return Arg.Is<Dictionary<string, string>>(d => d.Count == 0); }
         }
 
+        public static Dictionary<string, string> DictionaryWithSince
```

Comment:
yes indeed both `DictionaryWithApiOptions `and `DictionaryWithApiOptionsAndSince `are specific to  `GitsClient`, it doesn't feel right to put them in `Arg.cs`, should i define them in both `GistsClientTests `and `ObservableGistsTests `or remove them entirely and use the expression each time ?